pkg/windows/windows.go

```diff
@@ -534,28 +534,11 @@ func (vm *windows) ConfigureAzureCloudNodeManager(nodeName string) error {
 	return nil
 }
 
-func (vm *windows) ConfigureKubeProxy(nodeName, hostSubnet string) error {
-	endpointIP, err := vm.Run(NetworkConfScriptPath, true)
-	if err != nil {
-		return errors.Wrap(err, "error creating HNS endpoint")
-	}
-
-	kubeProxyServiceArgs := "--windows-service --v=4 --proxy-mode=kernelspace --feature-gates=WinOverlay=true " +
-		"--hostname-override=" + nodeName + " --kubeconfig=c:\\k\\kubeconfig " +
-		"--cluster-cidr=" + hostSubnet + " --log-dir=" + kubeProxyLogDir + " --logtostderr=false " +
-		"--network-name=" + OVNKubeOverlayNetwork + " --source-vip=" + strings.TrimSpace(endpointIP) +
-		" --enable-dsr=false"
-
-	kubeProxyService, err := newService(KubeProxyPath, kubeProxyServiceName, kubeProxyServiceArgs,
-		[]string{HybridOverlayServiceName})
-	if err != nil {
-		return errors.Wrapf(err, "error creating %s service object", kubeProxyServiceName)
-	}
-
-	if err := vm.ensureServiceIsRunning(kubeProxyService); err != nil {
```

Comment:
Will the kube-proxy service still have the "OpenShift managed" description if we don't call  `ensureServiceIsRunning` anymore?

Reply:
Yes, [WICD does this as it is starting the service](https://github.com/openshift/windows-machine-config-operator/blob/0197a69eb52621ad57adea805cc63abaaf94e9d5/pkg/daemon/controller/controller.go#L262)